test/dynamo/test_hooks.py

```diff
@@ -582,11 +583,13 @@ def fn(x, y):
                     else contextlib.nullcontext()
                 )
                 with compiled_bwd_ctx:
-                    b = torch.zeros(3, requires_grad=True)
+                    b = torch.tensor([2.0, 2.0, 2.0], requires_grad=True)
                     x.backward(b)
                     self.assertEqual(cnts.frame_count, 1)
                     # X goes to x*2 becaue of mul_
                     self.assertEqual(x, torch.tensor([0.5, 0.5, 0.5]) * 2)
+                    # This test proves grad aliasing works -
+                    self.assertEqual(x.grad, b * 5)
```

Comment:
I cannot easily tell that the equation here is correct. It would be a lot more convincing if you compared the result here to eager mode.